### PR TITLE
Update Modal integration tile

### PR DESCRIPTION
**Pull request by irfansharif**
- Update the default dashboard.
- Update the product hero (we made some changes to our UI).
- Update the README to mention that we're not (yet) collecting metrics.

### What does this PR do?

We're now also collecting function logs. Update things to reflect as much.

### Motivation

See above.

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [/] Feature or bugfix has tests
- [x] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)
- [/] If this PR includes a log pipeline, please add a description describing the remappers and processors. 

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Comment by irfansharif** [comment]
+cc @pawalt.

**Review by emarsha94 — APPROVED** [review]

File: modal/README.md [hunk 1]
```diff
@@ -27,19 +27,19 @@ Logs and metrics from your Modal apps should now start appearing in Datadog.
...
-Modal collects audit logs and all application logs (all stdout/stderr logging).
+Modal collects all application (stdout/stderr logging for functions) and audit logs.
 
 ### Events
 
 Modal does not include any events.
 
+### Metrics
+
+Modal does not collect any metrics (yet).
```
Comment (aliciascott) [inline modal/README.md:40]:
```suggestion
There are no metrics collected for this integration.
```
For this we suggest just to state there are no metrics collected.

**Review by aliciascott — CHANGES_REQUESTED** [review]
Hi @irfansharif just a minor suggestion, please let me know once updated and I'll approve :)

**Review by irfansharif — COMMENTED** [review]
Done!

**Review by emarsha94 — APPROVED** [review]

File: modal/images/modal-product-hero.jpg
Comment (emarsha94) [inline modal/images/modal-product-hero.jpg:1]: If you are updating tile images, you will need to add this to the media carousel in your manifest.json https://docs.datadoghq.com/developers/integrations/create_a_tile/?tab=buildatileontheintegrationspage#media-carousel

**Review by emarsha94 — CHANGES_REQUESTED** [review]

**Review by aliciascott — APPROVED** [review]

File: modal/images/modal-product-hero.jpg
Comment (emarsha94) [reply modal/images/modal-product-hero.jpg:1]: @irfansharif please see above

**Review by emarsha94 — APPROVED** [review]

**Review by moreauowen — APPROVED** [review]

**Pull request merged**